### PR TITLE
Add issue and PR templates and CODEOWNERS file

### PR DESCRIPTION
**Pull request by SebaLukas**
- Added issue and PR templates
- Added CODEOWNERS file

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+name: Bug Report
+description: Thanks for taking the time to fill out this bug report!
```
Comment (barsnick) [inline .github/ISSUE_TEMPLATE/bug_report.yaml:2]:
```suggestion
description: Thanks for taking the time to fill out this bug report!
label: [bug]
```
See https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository

File: .github/ISSUE_TEMPLATE/feature_request.yaml [hunk 2]
```diff
@@ -0,0 +1,27 @@
+name: Feature Request
+description: Thanks for taking the time to fill out this feature request!
```
Comment (barsnick) [inline .github/ISSUE_TEMPLATE/feature_request.yaml:2]:
```suggestion
description: Thanks for taking the time to fill out this feature request!
label: [enhancement]
```
`enhancement` is a default label, see https://docs.github.com/en/issues/using-labels-and-milestones-to-track-work/managing-labels

**Review by barsnick — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/bug_report.yaml (hunk 1 above)
Comment (SebaLukas) [reply .github/ISSUE_TEMPLATE/bug_report.yaml:2]: Done

File: .github/ISSUE_TEMPLATE/feature_request.yaml (hunk 2 above)
Comment (SebaLukas) [reply .github/ISSUE_TEMPLATE/feature_request.yaml:2]: Done

File: .github/ISSUE_TEMPLATE/bug_report.yaml (hunk 1 above)
Comment (barsnick) [reply .github/ISSUE_TEMPLATE/bug_report.yaml:2]: Looks good, untested. Let's push.

**Review by barsnick — APPROVED** [review]

**Pull request merged**